src/main/java/pokecube/pokeplayer/network/handlers/AttackLocation.java

```diff
@@ -61,15 +63,15 @@ public void handleCommand(IPokemob pokemob)
         else
         {
             // Do default behaviour.
-            pokecube.core.interfaces.pokemob.commandhandlers.AttackLocationHandler defaults = new pokecube.core.interfaces.pokemob.commandhandlers.AttackLocationHandler();
+            AttackLocationHandler defaults = new AttackLocationHandler();
             ByteBuf buffer = Unpooled.buffer();
             this.writeToBuf(buffer);
             defaults.readFromBuf(buffer);
             defaults.handleCommand(pokemob);
         }
     }
 
-    /*@Override
+    @Override
     public void writeToBuf(ByteBuf buf)
     {
         super.writeToBuf(buf);
```

Comment:
you need to write location to buf here as well, see AttackLocationHandler for how to do that